imports/plugins/core/payments/server/no-meteor/mutations/enablePaymentMethodForShop.js

```diff
@@ -0,0 +1,36 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import SimpleSchema from "simpl-schema";
+import { Shops } from "/lib/collections";
+import { paymentMethods as allPaymentMethods } from "/imports/plugins/core/core/server/no-meteor/pluginRegistration";
+
+const paramsSchema = new SimpleSchema({
+  isEnabled: Boolean,
+  paymentMethodName: String,
+  shopId: String
+});
+
+export default async function enablePaymentMethodForShop(context, input = {}) {
+  paramsSchema.validate(input);
+  const { isEnabled, paymentMethodName, shopId } = input;
+
+  if (!allPaymentMethods[paymentMethodName]) {
+    throw new ReactionError("not-found", "Requested payment method is invalid");
+  }
+
+  const shop = await context.queries.shopById(context, shopId);
+  const methods = new Set(shop.availablePaymentMethods);
+
+  if (isEnabled) {
+    methods.add(paymentMethodName);
+  } else {
+    methods.delete(paymentMethodName);
+  }
+
+  await Shops.update(
+    { _id: shop._id },
+    { $set: { availablePaymentMethods: Array.from(methods) } },
+    { bypassCollection2: true }
+  );
```

Comment:
`bypassCollection2` is only needed for Meteor collections, and generally only should be used in migration code. That bypasses checking the modifier against the schema that is attached to the collection.

Also, in the version of mongo driver we use, `update` is deprecated. So use `updateOne` instead.